projects/listening-comp/services/transcript-processor/app.py

```
@@ -17,6 +25,42 @@
 OUTPUT_DIR = os.environ.get('OUTPUT_DIR', '/shared/data')
 os.makedirs(OUTPUT_DIR, exist_ok=True)
 
+# Initialize ServiceOrchestrator
+service_orchestrator = ServiceOrchestrator(device=DEVICE)
```

Ruff (0.8.2)

Undefined name `DEVICE`

(F821)